grpcbp/server_middlewares_test.go

```diff
@@ -186,18 +187,18 @@ func (t *mockService) PingStream(c pb.TestService_PingStreamServer) error {
 
 func TestInjectPrometheusUnaryServerClientInterceptor(t *testing.T) {
 	const (
-		serviceName = "testSvc"
-		serverName  = "testServer"
-		method      = "Ping"
+		localService      = "testSvc"
+		remoteServiceSlug = "testServer"
```

Comment:
maybe it would be clear if you kept the variable names but swapped the values to to

```suggestion
		serviceName = "com.example.reddit.preference.PreferenceService"
		serverName  = "example-preference-server"
```